app/models/energy/ScheduleFile.py

```diff
@@ -0,0 +1,57 @@
+"""Schedule File Schema"""
+from pydantic import BaseModel, Schema, validator, ValidationError, UrlStr, ConstrainedStr
+from typing import List, Union
+from enum import Enum
+from uuid import UUID, uuid4
+from datetime import datetime
+from app.models.energy.ScheduleBase import UnitType
+
+
+class ScheduleFile(BaseModel):
+    """A list of values used to create a .csv file for the entire year or a period of
+  analysis within the year."""
+
+    type: Enum('ScheduleFile', {'type': 'ScheduleFile'})
+
+    name: str = Schema(
+        ...,
+        regex=r'^[\s.A-Za-z0-9_-]*$',
+    )
+
+    start_day_of_year: int = Schema(
+        ...,
+        ge=1,
+        le=366,
+        description='Used to define the start day for the analysis period.'
+    )
+
+    values: List[int] = Schema(
+        ...,
+        minItems=24,
```

Comment:
This looks like it might translate to [ScheduleFixedInterval](https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/model/ScheduleFixedInterval.hpp) rather than [ScheduleFile](https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/model/ScheduleFile.hpp) in OpenStudio.  ScheduleFixedInterval has all the values embedded in the OSM where ScheduleFile refers to an external file.

Reply:
I didn't realize that is what ScheduleFixedInterval does.  On [which of the properties](https://openstudio-sdk-documentation.s3.amazonaws.com/cpp/OpenStudio-2.8.0-doc/model/html/classopenstudio_1_1model_1_1_schedule_fixed_interval.html) do you specify the list of 8760 values?

Reply:
Its in the base class `ScheduleInterval`, the methods is `setTimeSeries` https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/model/ScheduleInterval.hpp#L72

Reply:
TimeSeries class can go back and forth with an OpenStudio::Vector which is basically an array of floats https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/utilities/data/TimeSeries.hpp

Reply:
Thanks for all of the info @macumber . ScheduleInterval it is!

Reply:
@chriswmackey should I add the `start_day_of_year` as `start_month` and `start_day` the way it is described in OpenStudio?